### PR TITLE
Dynamically render avatar on header, and escape single quotes in when adding items to collection

### PR DESCRIPTION
**Pull request by SMbrobot10**

### CODE REVIEW

File: client/Components/Header/Header.jsx [hunk 1]
```diff
@@ -79,17 +102,46 @@ const Header = ({ loggedIn, setLoggedIn }) => {
...
-            </Link>
-            <Dropdown value="selection" onChange={(e) => handleChange(e)}>
-              <option key={userOptions[0].value} value={userOptions[0].value}>
+            {/* <Link to="/"> */}
+            <div
+              style={{
+                display: "flex",
+                alignItem: "center",
+              }}
+            >
```
Comment (Mariella-Arias) [inline client/Components/Header/Header.jsx]: For consistency, we should make all styling attributes using styled-components

File: client/Components/Header/Header.jsx [hunk 2]
```diff
@@ -79,17 +102,46 @@ const Header = ({ loggedIn, setLoggedIn }) => {
...
               </option>
               <option key={userOptions[1].value} value={userOptions[1].value}>
                 {userOptions[1].label}
               </option>
-            </Dropdown>
+              <option key={userOptions[2].value} value={userOptions[2].value}>
+                {userOptions[1].label}
+              </option> */}
+              </Dropdown>
+            </div>
```
Comment (Mariella-Arias) [inline client/Components/Header/Header.jsx]: It's good practice to only have working code in the main branch. Let's get rid of any unused code here and console logs as well.

File: client/Components/Homepage/Homepage.jsx [hunk 3]
```diff
@@ -30,6 +33,7 @@ const Homepage = ({
         },
       })
       .then((collection) => {
+        console.log("COLLECTION: ", collection);
         setUserCollection(() => collection.data.rows[0]);
```
Comment (Mariella-Arias) [inline client/Components/Homepage/Homepage.jsx:37]: Let's remove console logs used in development.

File: server.js [hunk 4]
```diff
@@ -100,7 +98,23 @@ app.post("/signup", (req, res) => {
...
+                algorithm: "HS256",
+              }
+            );
+            res.cookie("token", token);
+            res.status(200).send(response);
+          })
+          .catch((err) => {
+            console.log("Error creating user", err);
+            res.status(500).send(err);
+          });
```
Comment (Mariella-Arias) [inline server.js]: For consistency let's get rid of nested promises.

File: server.js [hunk 5]
```diff
@@ -235,12 +249,14 @@ app.post(`/saveItems`, (req, res) => {
...
   saveItemToDB(itemData)
     .then((response) => {
       res.status(200).send(response);
       console.log(`Success saving item to database`);
     })
     .catch((err) => {
+      console.log("error sending items,", err);
       res.status(500).send(err);
       console.log(`Error saving item to database`);
     });
```
Comment (Mariella-Arias) [inline server.js]: Remove console logs before merging to main branch.

File: server.js [hunk 6]
```diff
@@ -367,6 +382,13 @@ app.get("/username/collectionValue", (req, res) => {
       res.status(500).send(err);
     });
 });
+//   //   });
+//   // });
+// });
+app.use(
+  "/resources/mario.jpg",
+  express.static(path.join(__dirname, "resources", "mario.jpg"))
+);
```
Comment (Mariella-Arias) [inline server.js:383]: Let's remove unused code before merging.

**Review by Mariella-Arias — CHANGES_REQUESTED** [review]
Looks great! Let's just keep the code consistent and clean. Check out the few comments I made and should be good for merging after that.

File: server.js (hunk 6 above)
Comment (SMbrobot10) [reply server.js:383]: This is used when signing in as the default avatar assignment to the users

**Pull request closed without merging**